### PR TITLE
Revert changes to recover nyan-cat-formatter and use UnicornFormatter

### PR DESCRIPTION
**Pull request by mdeniz**
@adrianschroeter please review it.
I saw you in other PRs lowering/changing the version for some gems, please be sure that this doesn't revert any of your changes :smile:

### CODE REVIEW

File: src/api/spec/spec_helper.rb [hunk 1]
```diff
@@ -73,10 +73,8 @@
   Kernel.srand config.seed
 
   # Custom formatter, but not in Travis-CI and neither during package build
-  begin
-    require 'nyan_cat_formatter'
+  unless ENV['TRAVIS']
```
Comment (adrianschroeter) [inline src/api/spec/spec_helper.rb]: It must be also disabled in building packages

File: src/api/Gemfile.lock [hunk 2]
```diff
@@ -152,7 +152,7 @@ GEM
       mime-types-data (~> 3.2015)
     mime-types-data (3.2016.0521)
     mini_portile2 (2.1.0)
-    minitest (5.9.0)
+    minitest (5.8.3)
```
Comment (adrianschroeter) [inline src/api/Gemfile.lock]: nope ... would break build again

File: src/api/Gemfile.lock [hunk 3]
```diff
@@ -226,6 +228,10 @@ GEM
     responders (2.2.0)
       railties (>= 4.2.0, < 5.1)
     riddle (1.5.12)
+    rspec (3.5.0)
+      rspec-core (~> 3.5.0)
+      rspec-expectations (~> 3.5.0)
+      rspec-mocks (~> 3.5.0)
```
Comment (adrianschroeter) [inline src/api/Gemfile.lock]: hm, where is this block coming from?

Comment (mdeniz) [inline src/api/Gemfile.lock]: I just recreate my vagrant machine from the scratch, rebase to master, and running `vagrant exec bundle install`... it appeared

File: src/api/Gemfile.lock (hunk 2 above)
Comment (mdeniz) [inline src/api/Gemfile.lock]: Then we should fix the version in the Gemfile

File: src/api/spec/spec_helper.rb (hunk 1 above)
Comment (mdeniz) [inline src/api/spec/spec_helper.rb]: How?

File: src/api/Gemfile.lock (hunk 2 above)
Comment (adrianschroeter) [inline src/api/Gemfile.lock]: On Dienstag, 30. August 2016, 08:10:01 CEST wrote Mois�s D�niz Alem�n:
> > @@ -152,7 +152,7 @@ GEM
> >        mime-types-data (~> 3.2015)
> >      mime-types-data (3.2016.0521)
> >      mini_portile2 (2.1.0)
> > -    minitest (5.9.0)
> > +    minitest (5.8.3)
> 
> Then we should fix the version in the Gemfile

right, my fault .... can you fix that also in this commit?

-- 

Adrian Schroeter
email: adrian@suse.de

SUSE Linux GmbH, GF: Felix Imend�rffer, Jane Smithard, Graham Norton, HRB 21284 (AG N�rnberg) 
 
Maxfeldstra�e 5                         
90409 N�rnberg 
Germany

**Comment by bgeuken** [comment]
@mdeniz Please don't merge pull requests that are not approved by someone else.

So, right now our OBS packages are in state unresolved due to missing nyan cat packages: https://build.opensuse.org/package/show/OBS:Server:Unstable/obs-server
Can you provide them please or should we revert this one again?:-)

**Comment by mdeniz** [comment]
I will revert it. But is RSpec versioin 3.5.0, not the nyan cat package....

**Pull request merged**